clusters/.sops.yaml

```diff
@@ -0,0 +1,4 @@
+creation_rules:
+  - path_regex: .*.yaml
+    encrypted_regex: ^(data|stringData)$
+    pgp: BFAD1CA2762581D260113596323CD3AC9A04F637
```

Comment:
where is this coming from? is there a CLI to generate this rnd str? do we want to store this as an ENV?

Reply:
is it the GPG key?